appveyor.yml

```diff
@@ -10,6 +10,13 @@ environment:
 
 version: "$(major).$(minor).$(release).{build}"
 
+dotnet_csproj:
+  patch: true
+  file: '**\*.csproj'
+  assembly_version: '$(app_version).$(build_number)'
+  file_version: '$(app_version).$(build_number)'
+  informational_version: '$(app_version).$(build_number)'
+
```

Comment:
@msevestre this didn't work for the FuncParser.dll (.net standard). Any idea how to fix?

Reply:
is it a donet csproj project?
or is it still an older format ?

Reply:
ok it is one. But the versions are all missing from the project file!
https://github.com/Open-Systems-Pharmacology/OSPSuite.FuncParser/blob/master/src/OSPSuite.FuncParser/OSPSuite.FuncParser.csproj

Please see for example OSPSuite.Core to see how to define the versions